.gitignore

```diff
@@ -130,3 +130,11 @@ dist/
 .awcache
 
 .pnp.*
+
```

Comment:
Unless we have good reasons we don't need to save individual code-editor configs I believe.  
@bumi you added `.nova` further abov. What kind of file is that? Just asking out of cuirosity.

Reply:
I had tried https://nova.app/ editor. .nova I think is their editor config file.

Reply:
Ah thanks, tried to figure out what it was but too many "nova"s out there.  Will add it to the editor section.